### PR TITLE
OSDOCS-4176: fix typos

### PR DESCRIPTION
**Pull request by jldohmann**
Version(s): 4.10+

Issue: https://issues.redhat.com/browse/OSDOCS-4176

Link to docs preview: http://file.rdu.redhat.com/jdohmann/OSDOCS-4176/networking/hardware_networks/using-pod-level-bonding.html#nw-sriov-cfg-creating-bond-network-attachment-definition_using-pod-level-bonding

QE review: N/A (typo fix)

Additional information:

### CODE REVIEW

File: modules/nw-sriov-cfg-bond-interface-with-virtual-functions.adoc [hunk 1]
```diff
@@ -35,7 +35,7 @@ apiVersion: "k8s.cni.cncf.io/v1"
       "type": "bond", <1>
       "cniVersion": "0.3.1",
       "name": "bond-net1",
-      "ifname: "bond0" <2>
+      "ifname": "bond0" <2>
```
Comment (snarayan-redhat) [inline modules/nw-sriov-cfg-bond-interface-with-virtual-functions.adoc:38]: Need a `,` here too

**Review by snarayan-redhat — COMMENTED** [review]
looks good. Just one small comment.

File: modules/nw-sriov-cfg-bond-interface-with-virtual-functions.adoc (hunk 1 above)
Comment (jldohmann) [reply modules/nw-sriov-cfg-bond-interface-with-virtual-functions.adoc]: good catch, thank you. fixed

**Pull request merged**